src/server.ts

```diff
@@ -5,28 +5,45 @@ import { Client } from "pg";
 import { getEnvVarOrFail } from "./support/envVarUtils";
 import { setupDBClientConfig } from "./support/setupDBClientConfig";
 
-dotenv.config(); //Read .env file lines as though they were env vars.
+dotenv.config();
 
 const dbClientConfig = setupDBClientConfig();
 const client = new Client(dbClientConfig);
 
-//Configure express routes
 const app = express();
 
-app.use(express.json()); //add JSON body parser to each following route handler
-app.use(cors()); //add CORS support to each following route handler
+app.use(express.json());
+app.use(cors());
 
 app.get("/", async (_req, res) => {
-    res.json({ msg: "Hello! There's nothing interesting for GET /" });
+    try {
+        const allHistory = await client.query("SELECT * FROM pastes;");
```

Comment:
while we haven't reviewed the front end yet so not 100% sure how its working, we would perform the filtering and sorting in this SQL query to simplify the front end code and reduce how much data the server is requesting.